common/pb/admin_commands.proto

```diff
@@ -26,3 +27,13 @@ message Command_ReloadConfig {
 		optional Command_ReloadConfig ext = 1002;
 	}
 }
+
+message Command_AdjustMod {
```

Comment:
So are you basically wanting me to create its own response container and move the failed event's from above into the response container (in a nutshell)?  The successful events for the sending admin could be also moved , but that seems silly if the design of the notification for this type of command is more of a broadcast to staff members.